app/steps/personal_detail.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class IntroductionIntroduceYourself < Step
+class PersonalDetail < Step
```

Comment:
for some reason, visit a URL slug of `personal-information` feels sketchier than `personal-detail` to me. But could just be me! thoughts from @alanjosephwilliams ?